stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -243,8 +243,7 @@ private static String mapFlagsToStringUsingAnnotation(
     }
   }
 
-  private static boolean isDefaultValue(
-      Float value) {
+  private static boolean isDefaultValue(Float value) {
```

Comment:
Strongly prefer undoing this pattern.  It increases the number of methods for very weak utility.